README.md

```diff
@@ -58,87 +58,87 @@ $ bower install validator-js
+| **isLatLong(str)**                            | check if the string is a valid latitude-longitude coordinate in the format `lat,long` or `lat, long`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| **isLength(str [, options])**                 | check if the string's length falls in a range.<br/><br/>`options` is an object which defaults to `{min:0, max: undefined}`. Note: this function takes into account surrogate pairs.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
```

Comment:
Can you avoid reformatting the entire README? there should only be a one line change, where you introduce the new locale.